modules/persistent-storage-hostpath-static-provisioning.adoc

```diff
@@ -10,27 +10,27 @@ A pod that uses a hostPath volume must be referenced by manual (static) provisio
 
 .Procedure
 
-. Define the persistent volume (PV). Create a file, `pv.yaml`, with the `PersistentVolume` object definition:
+. Define the persistent volume (PV) by creating a `pv.yaml` file with the `PersistentVolume` object definition:
 +
 [source,yaml]
 ----
-  apiVersion: v1
-  kind: PersistentVolume
-  metadata:
-    name: task-pv-volume <1>
-    labels:
-      type: local
-  spec:
-    storageClassName: manual <2>
-    capacity:
-      storage: 5Gi
-    accessModes:
-      - ReadWriteOnce <3>
-    persistentVolumeReclaimPolicy: Retain
-    hostPath:
-      path: "/mnt/data" <4>
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: task-pv-volume <1>
+  labels:
+    type: local
+spec:
+  storageClassName: manual <2>
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteOnce <3>
+  persistentVolumeReclaimPolicy: Retain
+  hostPath:
+    path: "/mnt/data" <4>
 ----
-<1> The name of the volume. This name is how it is identified by persistent volume claims or pods.
+<1> The name of the volume. This name is how the volume is identified by persistent volume claims or pods.
 <2> Used to bind persistent volume claim requests to this persistent volume.
```

Comment:
```suggestion
<2> Used to bind persistent volume claim requests to this PV.
```
Since you've already used "PV" to mean "persistent volume" in this section, consider using "PV" here.

Reply:
:heavy_check_mark: